flutter_secure_storage_web/lib/src/subtle.dart

```diff
@@ -18,12 +18,12 @@
 library common;
 
 import 'dart:convert' show jsonDecode;
-import 'dart:html';
 import 'dart:js_util' as js_util;
 import 'dart:typed_data';
 
 import 'package:flutter_secure_storage_web/src/jsonwebkey.dart' show JsonWebKey;
 import 'package:js/js.dart';
```

Comment:
The migration to the new js-interop approach also means getting rid of the `package:js`.

https://dart.dev/interop/js-interop/package-web

See also these comments https://github.com/firebase/flutterfire/issues/12027#issue-2038508266 from Kevin Moore (Flutter Web PM)